sandy/sections/mf6.py

```diff
@@ -0,0 +1,515 @@
+"""
+This module contains only two public functions:
+
+    * `read_mf6`
+    * `write_mf6`
+
+Function `read` reads a MF6/MT section from a string and produces a content
+object with a dictionary-like structure.
+The content object can be accessed using most of the keywords specified in
+the ENDF6 manual for this specific MF section.
+
+Function `write_mf6` writes a content object for a MF6/MT section into a
+string.
+MAT, MF, MT and line numbers are also added (each line ends with a `\n`).
+"""
+__author__ = "Aitor Bengoechea"
+__all__ = [
+        "read_mf6",
+        "write_mf6",
+        ]
+
+import sandy
+from sandy import zam
+
+
+def read_mf6(tape, mat, mt):
+    """
+    Parse MAT/MF=6/MT section from `sandy.Endf6` object and return
+    structured content in nested dictionary.
+
+    Parameters
+    ----------
+    tape : `sandy.Endf6`
+        endf6 object containing requested section.
+    mat : `int`
+        MAT number.
+    mt : `int`
+        MT number.
+
+    Returns
+    -------
+    out : `dict`
+        Content of the ENDF-6 tape structured as nested `dict`.
+
+    Examples
+    --------
+    Since the outputs are very large, I am only going to check the first energy
+    of each of the law-dependent structures:
+    Law 1:
```

Comment:
Can you use some markdown, like bold font:

**Law 1**

Reply:
I did in pycharm because spyder dont have markdowns